### PR TITLE
Coverity badge and sample script

### PR DESCRIPTION
**Pull request by plavjanik**
- Coverity badge, script to scan, and Jenkins pipeline for scanning
- Results are available at https://scan.coverity.com/projects/zowe-sample-spring-boot-api-service?tab=overview

### CODE REVIEW

File: coverity/scan.groovy [hunk 1]
```diff
@@ -0,0 +1,31 @@
+pipeline {
+    agent {
+        docker {
+            image 'plavjanik/coverity-oss'
```
Comment (JirkaAichler) [inline coverity/scan.groovy:4]: Would it make sense to move to `zowe` project instead of your name?

**Review by JirkaAichler — APPROVED** [review]
Nice job!

**Comment by JirkaAichler** [comment]
I have one more question. How the Jenkins script is executed when we are using CircleCI?

File: coverity/scan.groovy (hunk 1 above)
Comment (plavjanik) [reply coverity/scan.groovy:4]: I will bring it to Mark Ackert. I cannot do it myself.

**Comment by plavjanik** [comment]
Coverity has been added to CircleCI on the master.

**Pull request merged**